app/routes/bdb/components/BdbDetail.tsx

```diff
@@ -479,6 +502,25 @@ const BdbDetail = () => {
       </ContentSection>
 
       <Flex column gap="var(--spacing-md)" margin="var(--spacing-md) 0 0 0">
+        <Flex column gap="var(--spacing-sm)">
+          <Flex justifyContent="space-between" alignItems="center">
```

Comment:
```suggestion
          <Flex wrap justifyContent="space-between" alignItems="center">
```

How does this look on mobile? These should wrap I believe

Reply:
Looks a lot better with wrap on mobile:
![image](https://github.com/user-attachments/assets/a1b021ff-1606-408f-90a5-acf8f3b8abed)

Reply:
but does it currently wrap like this?

Reply:
no. The title and button was on the same line. With the wrap property, it looks like the above